shell/src/main/java/alluxio/cli/fsadmin/pathconf/RemoveCommand.java

```diff
@@ -69,19 +77,37 @@ public void validateArgs(CommandLine cl) throws InvalidArgumentException {
   @Override
   public int run(CommandLine cl) throws IOException {
     AlluxioURI path = new AlluxioURI(cl.getArgs()[0]);
+    boolean recursive = cl.hasOption(RECURSIVE_OPTION.getOpt());
     if (cl.hasOption(KEYS_OPTION_NAME)) {
       String[] keys = cl.getOptionValue(KEYS_OPTION_NAME).split(",");
       Set<PropertyKey> propertyKeys = new HashSet<>();
       for (String key : keys) {
         propertyKeys.add(PropertyKey.fromString(key));
       }
-      mMetaConfigClient.removePathConfiguration(path, propertyKeys);
+      handleRemove(path, recursive,
+          p -> mMetaConfigClient.removePathConfiguration(p, propertyKeys));
```

Comment:
this is a bit too convoluted. I think it's best to structure it as:

```java
Set<PropertyKey> keysToRemove = ...;
Set<AlluxioURI> pathsToRemove = isRecursive ? ... : emptySet();
if (keysToRemove.isEmpty()) {
  // remove all keys
  for (AlluxioURI p : pathsToRemove) {
    client.removePathConfiguration(p);
  }
  return 0;
}

for (AlluxioURI p : pathsToRemove) {
  client.removePathConfiguration(p, keysToRemove);
}

return 0;
```

Reply:
okay, i turn to refactor the code follow your style